photo-effects/src/components/Dashboard/Upload/upload.css

```diff
@@ -0,0 +1,26 @@
+
+
+.uploadBox {
```

Comment:
decrease the height a bit on the dropzone component. so users can at least partially see the projects component. right now the dropzone takes up the whole screen. to a new user, all they will see is the dropzone and may not think to scroll down.

Reply:
Yeah i was thinking about that and couldn't make up my mind. I'll lower the height i think you're right. Thank you! :)